### ISSUE TITLE
Fix doc permissions

### ISSUE THREAD

**Issue by nim-nim** [rpm-software-management/rpm#567]
A lot of effort has been expended to make attr setting unecessary in modern rpm. Unfortunately, it seems to have forgotten ` %doc`, I see files packaged with the 640 mask (as in the upstream archive) instead of 644.

**Comment by jayvdb** [comment]
It also appears to disregard any hard links, and doesnt re-hard-link them, which in my case is causing rpmlint to rage about file duplicates in the docs dir.

This is because doc and license are handled by `processSpecialDir` which is a bit special, most likely because it is using globs from the source and not obtaining files from the buildroot.

**Comment by pmatilai** [comment]
Agreed, rpm should ensure both %doc and %license files are always world-readable.

**Comment by jasontibbitts** [comment]
Is it possible to do this in a brp script?  I suppose what would work for files under %docdir, but things randomly marked as %doc would be missed.  As far as I know there's no way for brp scripts to see anything about file attributes (or really anything besides what's on the filesystem or passed on the command line).

**Comment by pmatilai** [comment]
The same goes for %license, with a slight difference: while documentation may sometimes be executable (example scripts), licenses never can. The two are closely related though so there's no point having separate tickets for the two, closing #1090 in favor of this.
